src/plugins/aws/config.ts

```
@@ -14,6 +14,22 @@ import { AwsConfig } from "./types";
 import { getDoc } from "firebase/firestore";
 import { sortBy } from "lodash";
 
+export const getFirstAwsConfig = async (authn: Authn) => {
```

i feel like this should throw an error if we're installed on multiple accounts. i'm worried people may get confused as to why certain behaviors work because we're just assuming a config without using an explicit account with `--account`

---

This is to grab the authentication method, e.g. IAM vs. IDC. It is assumed to be the same across all accounts.

Re: our previous discussion regarding this.